src/Organism/Sponsorship/index.js

```diff
@@ -0,0 +1,153 @@
+import React, { forwardRef } from 'react';
+import useSwr from 'swr';
+import List, { Item } from 'topo/Atom/List';
+import Box from 'topo/Atom/Box';
+import Divider from 'topo/Atom/Divider';
+import { Flex, Icon } from "@chakra-ui/core"
+import Skelly from 'topo/Atom/Skelly';
+import Text, { Heading, Link, CopyText } from 'topo/Atom/Text';
+import Content from 'topo/Molecule/Content';
+import { childrenOfType, makePureBox } from 'topo/_utils';
+import { useString, apiFetch } from 'topo/utils';
+
+// GraphQL Query, pulls data from the sponsorship programs area
+const query = `{
+  cms {
+    programs {
+      items {
+        sponsorPerks
+      }
+    }
+  }
+}`;
+
+const SponsorTable = () => {
+
+  // Pull data from GraphQL Query from above
+  const { data, error } = useSwr(
+    query,
+    apiFetch,
+    {
+      revalidateOnFocus: false,
+      revalidateOnReconnect: false,
+    }
+  );
+
+  // Let pulled data from GraphQL be set equal to the levels variable
+  const { levels } = data?.cms?.programs?.items[0]?.sponsorPerks || {};
+  
+  // DEBUG: Print straight to console to ensure data is being queried correctly
+  // console.log(levels);
+
+  // Return HTML
+  return (
+    <Box>
+      <Heading as="h1">Virtual CodeDay Sponsorships</Heading>
+      <Heading as="h4">{(new Date()).getFullYear()}-{(new Date()).getFullYear()+1} School Year</Heading>
```

Comment:
I think it's best to omit this part because it doesn't make sense for all programs. For example, does the 2020-2021 school year include CodeDay Labs in July 2020, or July 2021?

Reply:
I have updated it so that it no longer displays the date in the latest commit.